lib/govuk_app_config/govuk_nodes/carrenza_fetcher.rb

```diff
@@ -0,0 +1,26 @@
+require "open-uri"
+
+class GovukNodes
+  class CarrenzaFetcher
+    def hostnames_of_class(node_class)
+      instances_of_class(node_class).map do |instance|
+        instance.fetch("name")
+      end
+    end
+
+  private
+
+    def instances_of_class(node_class)
+      url = "http://puppetdb.cluster/v2/nodes?#{query_string(node_class)}"
```

Comment:
Can this be provided by a env.d variable or some such? Hard coded names make me nervous.

Reply:
It's hardcoded in the script I ported, but sure.  Do you have somewhere I can source it from in puppet, or should I hardcode it there?

Reply:
Or I can put it in hiera but I'm assuming it's somewhere already

Reply:
@deanwilson Updated.